sensors/gcp_message_sensor.py

```diff
@@ -46,11 +47,22 @@ def setup(self):
         Create connection and initialize Kafka Consumer.
         """
         self._logger.debug('[KafkaGCPMessageSensor]: Initializing consumer ...')
-        self._consumer = KafkaConsumer(*self._topics,
-                                       client_id=self._client_id,
-                                       group_id=self._group_id,
-                                       bootstrap_servers=self._hosts,
-                                       value_deserializer=lambda m: json.loads(m.decode('utf-8')))
+        if self.config.get('tls_enable', False):
+            self._consumer = KafkaConsumer(*self._topics,
+                                           client_id=self._client_id,
+                                           group_id=self._group_id,
+                                           bootstrap_servers=self._hosts,
+                                           value_deserializer=lambda m: json.loads(m.decode('utf-8')),
```

Comment:
The sensor was crashing, because of non-json or malformed json I'm guessing.  Need to change it to:

```
value_deserializer=lambda m: self._try_deserialize(m.decode('utf-8'))
```

Reply:
Thanks.  I change it.